DEVELOPER.md

```diff
@@ -35,6 +35,7 @@ Envoy Gateway is built using a [make][make]-based build system. Our CI is based
 * Run `make test` to run the golang tests.
 
 ### Running code linters
+* Run `make lint-deps` to install linter dependencies.
```

Comment:
not needed, each lint dependency installs its binary, also tested by cloning a new repo and running `make lint` which worked with no issues